docs/cookbook/recipe_persisting_filters.rst

```diff
@@ -73,16 +73,12 @@ Per Admin :
         services:
             app.admin.user:
                 class: AppBundle\Admin\UserAdmin
-                tags:
-                    -
-                        name: sonata.admin
-                        manager_type: orm
-                        filter_persister: filter_persister_service_id
                 arguments:
                     - ~
                     - AppBundle\Entity\User
                     - ~
-
+                tags:
+                    - { name: sonata.admin manager_type: orm filter_persister: filter_persister_service_id }
```

Comment:
Please rever to multiline style, this is probably too long for docs and will make a horizontal scroll appear. Also, you forgot a few commas in there :P